tests/classes/core/PKPComponentRouterTest.php

```diff
@@ -267,10 +272,13 @@ public function testRoute()
         Registry::set('request', $this->request);
         $user = new \PKP\user\User();
         Registry::set('user', $user);
+        $serviceEndpoint = $this->router->getRpcServiceEndpoint($this->request);
+        $handler = $serviceEndpoint[0];
+        $handler->addPolicy(new UserRolesRequiredPolicy($this->request), true);
         $this->router->route($this->request);
 
-        self::assertNotNull($serviceEndpoint = & $this->router->getRpcServiceEndpoint($this->request));
-        self::assertInstanceOf('\PKP\controllers\grid\notifications\NotificationsGridHandler', $handler = & $serviceEndpoint[0]);
+        self::assertNotNull($serviceEndpoint);
+        self::assertInstanceOf('\PKP\controllers\grid\notifications\NotificationsGridHandler', $handler);
```

Comment:
BTW, you can use `self::assertInstanceOf(\PKP\controllers\grid\notifications\NotificationsGridHandler::class, $handler);` -- I've been converting string-based forms (especially `is_a` into `instanceof`) when I find them.

Reply:
Yeah, it's also possible to use it on instances (e.g. `$instance::class`) =]